webapp/.eslintrc.json

```diff
@@ -632,5 +632,47 @@
         "onlyEquality": false
       }
     ]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.tsx", "**/*.ts"],
+      "extends": [
```

Comment:
This is not working properly, but I don't know why is not failing in mattermost-webapp. I'm investigating it.

Reply:
My fault I hadn't upgrade to eslint 6 in my repo.